src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo.png"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active apple"
+            >
+              APPLE
+            </a>
+            <div class="rectangle"></div>
+            <!-- Elemento retângulo abaixo do link -->
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link samsung"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link colorido laptopsecomputers"
+              data-qa="is-active"
```

Comment:
Avoid using spaces in class names. Instead of `laptopsecomputers`, use a hyphen or underscore to separate words, like `laptops-computers` or `laptops_computers`.